app/graph/crud.py

```diff
@@ -21,107 +18,120 @@
 
 # List of acceptable node labels and relationship types
 # Modify these to add constraints
-node_labels = ['Address','Geography','Person','Company','Event']
-relationship_types = ['LIVES_IN','USED_TO_LIVE_IN','WORKS_FOR','LOCATED_IN','KNOWS','ATTENDED']
-base_properties = ['created_by','created_time']
+node_labels = ['Address', 'Geography', 'Person', 'Company', 'Event']
+relationship_types = ['LIVES_IN', 'USED_TO_LIVE_IN', 'WORKS_FOR', 'LOCATED_IN', 'KNOWS', 'ATTENDED', 'FRIEND']
+
+# Used for validation to ensure they are not overwritten
+base_properties = ['created_by', 'created_time']
+
 
-### NODES
 # CREATE new node
 @router.post('/create_node', response_model=Node)
-async def create_node(label:str, node_attributes: dict, 
+async def create_node(label: str, node_attributes: dict,
                       current_user: User = Depends(get_current_active_user)):
     # Check that node is not User
     if label == 'User':
         raise HTTPException(
             status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-            detail=f"Operation not permitted, cannot create a User with this method.",
+            detail="Operation not permitted, cannot create a User with this method.",
             headers={"WWW-Authenticate": "Bearer"})
 
     # Check that node has an acceptable label
     if label not in node_labels:
         raise HTTPException(
             status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-            detail=f"Operation not permitted, node label is not accepted.",
+            detail="Operation not permitted, node label is not accepted.",
             headers={"WWW-Authenticate": "Bearer"})
 
     # Check that attributes dictionary does not modify base fields
     for key in node_attributes:
         if key in base_properties:
             raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-                                detail=f"Operation not permitted, you cannot modify those fields with this method.",
+                                detail="Operation not permitted, you cannot modify those fields with this method.",
                                 headers={"WWW-Authenticate": "Bearer"})
-    
+
     unpacked_attributes = 'SET ' + ', '.join(f'new_node.{key}=\'{value}\'' for (key, value) in node_attributes.items())
 
-    cypher = (f'CREATE (new_node:{label})\n'
-               'SET new_node.created_by = $created_by\n'
-               'SET new_node.created_time = $created_time\n'
-               f'{unpacked_attributes}\n'
-               'RETURN new_node, LABELS(new_node) as labels, ID(new_node) as id')
+    cypher = f"""
```

Comment:
I like this f-string better - it's much cleaner. Didn't know it worked without the parentheses... will remove the trailing parentheses.